ambari/security/kerberos/principals.rst

```diff
@@ -0,0 +1,32 @@
+|rbp-stack| and Kerberos Principals
+===================================
+
+Each service and sub-service in |rbp-stack| must have its own principal.
+A **principal** name in a given realm consists of a primary name and an instance name, in this case the instance name is the FQDN of the host that runs that service.
+As services do not log in with a password to acquire their tickets, their principal's authentication credentials are stored in a **keytab** file, which is extracted from the Kerberos database and stored locally in a secured directory with the service principal on the service component host.
+
+.. image:: /img/ambari/Principals_and_Keytabs.png
```

Comment:
HDP Cluster -> |rbpstack| cluster. Wait for decision on final abbreviation before changing.